packages/forms-web-app/src/pages/projects/_utils/get-application-data.js

```diff
@@ -6,6 +6,8 @@ const badDateToNull = (date) => (date === '0000-00-00' ? null : date);
 
 const add28DaysToDate = (date) => (date ? dayjs(date).add(28, 'days').toISOString() : null);
 
+const preserveLinebreaks = (text) => text.replace(/\r\n|\r|\n/g, '<br>');
```

Comment:
As we are using this in multiple places, can we move this to a common export and import everywhere?

Reply:
It's only used here at the moment, but I can see it being useful elsewhere.  Perhaps the other functions above (badDateToNull and add28DaysToDate) could do with being moved into a utils as well.  This was the reason I left this one in situ as well as these were already here.

Reply:
Should this be another ticket to consolidate these functions?

Reply:
I have created a new reusable function as requested.